android/src/main/java/capacitor/plugin/appsflyer/sdk/AppsFlyerPlugin.kt

```diff
@@ -86,18 +88,17 @@ class AppsFlyerPlugin : Plugin() {
                     subscribeForDeepLink(getDeepLinkListener())
                 }
             }
-            start(activity ?: context.applicationContext, null, object : AppsFlyerRequestListener {
-                override fun onSuccess() {
-                    val ret = JSObject()
-                    ret.put("res", "ok")
-                    call.resolve(ret)
-                }
 
-                override fun onError(p0: Int, p1: String) {
-                    call.reject(p1, p0.toString())
+            if (manualStart == false) {
+                Log.d("appsflyer_", "manual start: $manualStart")
```

Comment:
You print this log only when `manualStart` is false. 
Do we need this log at all? 
If so, the tag name should be `AppsFlyer_Capacitor`